README.md

```diff
@@ -1,76 +1,130 @@
 ![](workshops/static/amy-logo.png)
 
-[Software Carpentry](http://software-carpentry.org) is now running three events a week.
-That number could double by the end of 2015
-but only if we can
+[Software Carpentry](http://software-carpentry.org) is running three events
+a week.  The number could double by the end of 2015, but only if we can
 [streamline setup and organization](http://software-carpentry.org/blog/2014/12/plans-for-2015-workshop-organization.html).
-The goal of this project is to build
-a web-based workshop administration application using Django to do that.
-It is not intended to be visible to the general public,
-or even to instructors (at least initially --- we may add that later).
-Instead,
-the target audience is administrators,
-most of whom are non-programmers,
-who need to keep track of
-what workshops are being arranged,
-when they're supposed to occur,
-who's teaching what,
-and so on.
-
-To get started:
-
-1.  Install Django and its dependencies.
+The goal of this project is to build a web-based workshop administration
+application using Django to do that.  It is not intended to be visible to the
+general public, or even to instructors (at least initially — we may add that
+later).
+
+Instead, the target audience is administrators, most of whon are
+non-programmers, who need to keep track of what workshops are being arranged,
+when they're supposed to occur, who's teaching what, and so on.
+
+# Getting started
+
+1. Clone the repository:
+
+    ~~~
+    $ git clone https://github.com/swcarpentry/amy.git
+    $ cd amy
+    ~~~
+
+2. Install Django and its dependencies:
 
     ~~~
     $ sudo python3 -m pip install -r requirements.txt
     ~~~
 
-2. Install [Bower](http://bower.io/) - the tool that manages Amy's JavaScript and CSS dependencies:
+    If you're experienced Python programmer, feel free to create a Python3-compatible virtualenv for Amy and install dependencies from `requirements.txt`.
+
+3. Install [Bower](http://bower.io/) — the tool that manages Amy's JavaScript and CSS dependencies:
 
     ~~~
-    $ npm install -g bower
+    $ sudo npm install -g bower
```

Comment:
On Tue, May 12, 2015 at 01:20:23PM -0700, Piotr Banaszkiewicz wrote:
> **Note**: if you don't want to use `sudo`, you can install `bower`
> locally. You'll need to set up your `$PATH` correctly, though. Look
> [here](https://docs.npmjs.com/getting-started/fixing-npm-permissions#option-2-change-npm-s-default-directory-to-another-directory)
> for details.

Looks good to me.

> **Note**: if you don't want to use `sudo`, you can install `amy`
> into Python virtualenv. Look
> [here](https://virtualenv.pypa.io/en/latest/userguide.html) for
> details. Make sure you're using Python3-compatible virtualenv!

No need for a virtualenv here, since Python loads stuff from your
~/.local by default.  Just use ‘pip install --user …’ with the
appropriate pip *or* use a virtualenv.  I'm not sure where Anaconda's
pip puts things if you use the virtualenv's pip and --user, but I
expect they have some sane default.